src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -1,10 +1,80 @@
 package frc.robot.subsystems;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.util.FiringSolution;
+
+import static frc.robot.Constants.Shooter.*;
+import static frc.robot.UnitConversions.Shooter.*;
 
+/**
+ * Represents the robot's shooter
+ * 
+ * <p>
+ * The mechanism contains a single flywheel with a fixed hood and a
+ * pneumatically actuated "deflector" that redirects the shot to a
+ * more horizontal trajectory
+ */
 public class ShooterSubsystem extends SubsystemBase {
+
+    // Flywheel motors
+    private WPI_TalonFX flywheelLeader;
+    private WPI_TalonFX flywheelFollower;
+
+    private DoubleSolenoid deflectorSolenoid;
+
     public ShooterSubsystem() {
+        // Configure the flywheel motors
+        flywheelLeader = new WPI_TalonFX(LEADER_MOTOR_ID);
```

Comment:
Per previous comments, the next time you touch this code, move it into a shared function that passes in the ID and PID values so you're not duplicating your work and adding a opportunity for a typo bug